src/main/java/se/sundsvall/casedata/integration/db/listeners/ErrandListener.java

```diff
@@ -38,20 +36,23 @@ public ErrandListener(final IncomingRequestFilter incomingRequestFilter, @Lazy f
 	}
 
 	@PrePersist
-	private void beforePersist(final ErrandEntity errand) {
+	private void prePersist(final ErrandEntity errand) {
 		errand.setErrandNumber(generateErrandNumber(errand.getCaseType()));
 	}
 
 	@PostPersist
 	private void postPersist(final ErrandEntity errand) {
 		errand.setCreatedByClient(incomingRequestFilter.getSubscriber());
 		errand.setCreatedBy(incomingRequestFilter.getAdUser());
-		LOG.info("Created errand with errandNumber: {}. Subscriber: {}. AD-user: {}", errand.getErrandNumber(), incomingRequestFilter.getSubscriber(), incomingRequestFilter.getAdUser());
+		LOG.info("Created errand with errandNumber: {}. Subscriber: {}. AD-user: {}", errand.getErrandNumber(), incomingRequestFilter.getSubscriber(),
+			incomingRequestFilter.getAdUser());
 	}
 
-	@PostUpdate
-	private void beforeUpdate(final ErrandEntity errand) {
+	@PreUpdate
```

Comment:
You have to update the other listeners as well. Right?

No PostPersist/Update should exist.